bug-finding.ipynb

```diff
@@ -234,6 +243,13 @@
     "# Bug 2 analysis and response"
    ]
   },
+  {
+    "cell_type": "markdown",
```

Comment:
Yes, this is right, but it lacks a bit of specificity. In particular, it's necessary to use `t.lemma_` because `t` is a `Token` object, while `t.lemma_` is the string representation of the Token's lemma -- which we can compare to our string `'the'`.

Also, what is the correct answer for the observed frequency?